godtools/App/Services/Resources/ResourcesCache.swift

```diff
@@ -47,12 +53,31 @@ class ResourcesCache {
         return getSortedResources().filterForLessonTypes(andFilteredBy: additionalFilter)
     }
     
+    func getAllVisibleTools(andFilteredBy additionalFilter: ResourceFilter? = nil) -> [ResourceModel] {
+        let defaultVariantAbbreviations = getMetaTools().map { $0.attrDefaultVariant }
+        let defaultVariants = getToolsWithAbbreviations(defaultVariantAbbreviations)
+        
+        let resourcesExcludingVariants = getResourcesWithoutMetaToolVariants()
+        
+        let combinedResourcesAndDefaultVariants = resourcesExcludingVariants + defaultVariants
```

Comment:
does this take the default order into account for ordering the list? I'm not sure if you do that sorting elsewhere

Reply:
nm, I see the sorting happening in the next method